iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ReactiveCapabilityShape.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2016, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network;
+
+/**
+ * Base interface for <code>Generator</code> reactive capabilities shape limits.
+ * <pre>
+ *           U (kV) ^
+ *                     |
+ *                     |      +------------------+
+ *                     |     /|                 /|
+ *                     |    / |                / |
+ *                     *---*--+---------------* |
+ *                     | / |  |               |  |
+ *                     |/  |  . (P, Q, U)     |  |
+ *                     +---|--|---------------|--+
+ *                     |   |  +---------------|--+
+ *                     |   | /                | /
+ *                     |   |/                 |/
+ *                     +---|------------------+
+ *                     |
+ * <-------------------*--------------------------------> P (MW)
+ *                    /
+ *                   /
+ *                  / Q (MVaR)
+ *
+ * - P (MW): Active Power (Horizontal Axis)
+ * - Q (MVaR): Reactive Power (Depth Axis)
+ * - U (kV): Voltage (Vertical Axis)
```

Comment:
I think it is confusing to use U in the javadoc but V in the code. Should it not all be V?